### PR TITLE
💄 Added instance ID to about modal and restyled credentials docs link

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
Closes N8N-4342

### CODE REVIEW

**Review by alexgrozav — DISMISSED** [review]
Looking good.

File: packages/design-system/src/components/N8nNotice/Notice.vue [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default Vue.extend({
 	font-size: var(--font-size-2xs);
 	display: flex;
 	color: var(--custom-font-black);
-	margin: var(--spacing-s) 0;
+	margin: var(--notice-margin, var(--spacing-s) 0);
```
Comment (alexgrozav) [inline packages/design-system/src/components/N8nNotice/Notice.vue:105]: Nice! We should have variables with fallback values for all our components. Can you add a tech deb task for it?

**Review by alexgrozav — APPROVED** [review]
Nice work!

File: packages/design-system/src/components/N8nNotice/Notice.vue (hunk 1 above)
Comment (OlegIvaniv) [reply packages/design-system/src/components/N8nNotice/Notice.vue:105]: Of course! https://linear.app/n8n/issue/N8N-4465

**Comment by janober** [comment]
Got released with `n8n@0.193.0`

**Pull request merged**